fdbserver/fdbserver.actor.cpp

```diff
@@ -619,6 +625,10 @@ static void printUsage( const char *name, bool devhelp ) {
 			   "                 Maximum amount of memory used for storage. The default\n"
 			   "                 value is 1GiB. When specified without a unit, MB is\n"
 			   "                 assumed.\n");
+		printf("  --cache_memory SIZE\n"
```

Comment:
So this help will only be displayed if you call `fdbserver --dev-help` - but not if you only pass `--help`. I assume the reason you want this to be a parameter is because users shouldn't be expected to touch knobs...

So I guess my confusion is: what exactly is the rule for putting this in `--dev-help`? Or should this be moved above this `if`-block? I have the same question for the `--memory` and `--storage_memory`  options...

Reply:
I honestly don't really know the intended distinction between `--dev-help` and `--help`, but I decided not to take any stand on that here. I added this option where it is because it seemed conceptually similar to the other two memory options you listed.

Reply:
I guess this makes sense - although I think none of them should be under `--dev-help` - though I think this is a bit out of scope for this PR. I will make an issue.